bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -496,7 +506,7 @@ private EnhancedRendition getBestRendition(@NotNull Asset asset, int width) thro
         }
         // If no rendition was found, attempt to use original
         if (bestRendition == null) {
-            bestRendition = new EnhancedRendition(asset.getOriginal());
+            bestRendition = getOriginal(asset);
```

Comment:
This will cause the `original` rendition to be filtered twice: once in `getOriginal` and another time down bellow.

Reply:
good finding, fixed it.